sys/src/9/386/pci.c

```diff
@@ -19,6 +19,8 @@
 
 #include "io.h"
 
+int pcicapoff(Pcidev *p);
```

Comment:
sorry, I should have mentioned this, but the more comment plan 9 coding standard looks like this:
int
pcicapoff(etc. etc.)

The return type on a line by itself. This is inherited style but we might as well try to be consistent.